src/CalculatorUITestFramework/HistoryPanel.cs

```diff
@@ -0,0 +1,51 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
We'll need copyright headers in all these files.

(We should set up StyleCopAnalyzers, which will automatically enforce this and help you add them.)